boards/fox/include/periph_conf.h

```diff
@@ -70,30 +73,30 @@ static const timer_conf_t timer_config[] = {
  * @brief UART configuration
  * @{
  */
-#define UART_NUMOF          (1U)
-#define UART_0_EN           1
-#define UART_1_EN           0
-#define UART_IRQ_PRIO       1
-
-/* UART 0 device configuration */
-#define UART_0_DEV          USART2
-#define UART_0_CLKEN()      (RCC->APB1ENR |= RCC_APB1ENR_USART2EN)
-#define UART_0_IRQ          USART2_IRQn
+static const uart_conf_t uart_config[] = {
+    /* device, rx pin, tx pin, bus, rcc bit, IRQn */
+    {
+        USART2,
+        GPIO_PIN(PORT_A, 3),
+        GPIO_PIN(PORT_A, 2),
+        RCC_APB1ENR_USART2EN,
```

Comment:
Can you adapt the description in line 77? IMO "bus" is not the correct description. Maybe something like bus-clock-enable or similar?